beta/src/content/learn/start-a-new-react-project.md

```diff
@@ -46,15 +46,18 @@ npm start
 
 </TerminalBlock>
 
-For more information, [check out the official guide.](https://create-react-app.dev/docs/getting-started)
-
-> Create React App doesn't handle backend logic or databases. You can use it with any backend. When you build a project, you'll get a folder with static HTML, CSS and JS. Because Create React App can't take advantage of the server, it doesn't provide the best performance. If you're looking for faster loading times and built-in features like routing and server-side logic, we recommend using a framework instead.
+For more information, [check out the official Vite docs.](https://vitejs.dev/)
 
 ### Popular alternatives {/*toolkit-popular-alternatives*/}
 
-* [Vite](https://vitejs.dev/guide/)
+Vite is not the only option for bundling your React-based single page applications. Here are some other popular options:
+
+* [create-react-app](https://create-react-app.dev/)
 * [Parcel](https://parceljs.org/getting-started/webapp/)
 
+> Vite's default templates don't handle backend logic or databases. You can use it with any backend. When you build a project, you'll get a folder with static HTML, CSS and JS. Because this template can't take advantage of the server, it doesn't provide the best performance. If you're looking for faster loading times and built-in features like routing and server-side logic, we recommend using a framework instead.
+
+
 ## Building with a full-featured framework {/*building-with-a-full-featured-framework*/}
 
 If you're looking to **start a production-ready project,** [Next.js](https://nextjs.org/) is a great place to start. Next.js is a popular, lightweight framework for static and server‑rendered applications built with React. It comes pre-packaged with features like routing, styling, and server-side rendering, getting your project up and running quickly. 
```

Comment:
Should it include Remix as an option as well?

Reply:
Remix should. A framework/library shouldn't really pick a favourite. Plus most people don't trust Vercel...

Plus has anyone thought about NX here. It's neutral on all this

Reply:
Why don't people trust Vercel @Jordan-Hall  ?

Reply:
Vercel is too good